_data/blogs.yml

```diff
@@ -7,7 +7,6 @@
   title: Interview with a teacher
   img: samyakbvs.jpeg
   author_name: Samyak Jain
-
```

Comment:
Why removed this line?

Reply:
whether I do not delete the line ?

Reply:
@anshuman73 I think this message of yours don't important. He has add that space again. Didn't?